### PR TITLE
Added Minio.io configuration steps to run as backend for restic.

### PR DESCRIPTION
**Pull request by koolhead17**

### CODE REVIEW

File: doc/Manual.md [hunk 1]
```diff
@@ -357,6 +357,29 @@ You can then easily initialize a repository that uses your Amazon S3 as a backen
 For an S3-compatible repository without TLS available, use the alternative URI
 protocol `s3:http://server:port/bucket_name`.
 
+# Create a Minio Server repository
```
Comment (fd0) [inline doc/Manual.md]: I think this would fit better as a sub-heading to the s3 backend example, could you add another `#`?

File: doc/Manual.md [hunk 2]
```diff
@@ -357,6 +357,29 @@ You can then easily initialize a repository that uses your Amazon S3 as a backen
...
+
+[Minio](https://www.minio.io) is an Open Source Object Storage, written in Go and compatible with AWS S3 API.
+
+### Pre-Requisites
+
+* Download and Install [Minio Client](https://minio.io/download/) & [Minio Server](https://minio.io/download/).
```
Comment (fd0) [inline doc/Manual.md]: Does a user really have to download the client?

**Comment by fd0** [comment]
Thanks for adding this!

File: doc/Manual.md (hunk 1 above)
Comment (koolhead17) [inline doc/Manual.md]: sure

File: doc/Manual.md (hunk 2 above)
Comment (koolhead17) [inline doc/Manual.md]: sure. Will clean the client part. :)

**Comment by fd0** [comment]
Merged, sorry for the delay.

**Pull request merged**